.github/workflows/tests.yml

```
@@ -221,6 +221,8 @@ jobs:
           exit 1
         fi
 
+        sleep 10
```

```suggestion
        time kubectl wait -n ${operatorNamespace} pod --selector=control-plane=harbor-operator --timeout=300s
```
It seems possible to use this

---

前面已经 wait deployment 了，此处怀疑是 harbor-operator 即便都已经部署成功了，webhook 也需要点时间才能生效，所以简单的 sleep

---

Not that easy to understand the response, but I agree with @cndoit18 's suggestion

---

> 前面已经 wait deployment 了，此处怀疑是 harbor-operator 即便都已经部署成功了，webhook 也需要点时间才能生效，所以简单的 sleep

@bitsf next time, you should leave an English message.

And will you do change for that part?

---

This `sleep 10` is just to workaround CI failed sometimes recently, because webhook seems not started even we already using `kubectl wait deployment`